src/shared/modules/editor/editorDuck.js

```diff
@@ -33,7 +33,9 @@ export const NOT_SUPPORTED_URL_PARAM_COMMAND = `${NAME}/NOT_SUPPORTED_URL_PARAM_
 // Supported commands
 const validCommandTypes = {
   play: (cmdchar, args) => `${cmdchar}play ${args.join(' ')}`,
-  cypher: (_, args) => args.join('\n')
+  cypher: (_, args) => args.join('\n'),
+  param: (cmdchar, args) => `${cmdchar}param ${args.join(' ')}`,
+  params: (cmdchar, args) => `${cmdchar}params ${args.join(' ')}`
```

Comment:
do we need to worry about interoperability here? can `:play` accept params? Also what happens if provided params are malformed/invalid?